build.sbt

```diff
@@ -5,8 +5,10 @@ name := "Agora"
 
 organization := "org.broadinstitute"
 
-scalaVersion := "2.11.12"
-dependencyOverrides += "org.scala-lang" % "scala-compiler" % "2.11.12"
+scalaVersion := "2.12.4"
+
+// uncomment to see compile warnings
+// scalacOptions := Seq("-unchecked", "-deprecation", "-feature")
```

Comment:
leaving off for now because warnings seem to be benign, but verbose. We can tackle quelling the warnings after we are successfully on 2.12.